core_backend/app/auth/config.py

```diff
@@ -1,11 +1,14 @@
 import os
 
+# user 1
 USER1_USERNAME = os.environ.get("USER1_USERNAME", "user1")
 USER1_PASSWORD = os.environ.get("USER1_PASSWORD", "fullaccess")
+USER1_RETRIEVAL_KEY = os.environ.get("USER1_RETRIEVAL_KEY", "user1-key")
```

Comment:
Should these be in the Db now? And we just get from there?

Reply:
Yeah, will add this to final PR once auth reads from DB.